packages/cli/src/index.ts

```diff
@@ -100,9 +100,14 @@ cli
       await startDevServer();
 
       const exitProcess = async () => {
```

Comment:
shall we add a timeout to force exit after 5 seconds?

Reply:
5 seconds too too long

Reply:
Welcome your pr 😀